files/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "startbootstrap-agency",
+  "repo": "leonciokof/startbootstrap-agency",
+  "files": {
+    "include": ["dist/css/agency.css", "dist/css/agency.min.css", "dist/js/agency.js", "dist/js/agency.min.js"]
```

Comment:
`"basePath": "dist"` and drop the dist in all the paths. Also can you only include the minified versions